app/furniture/marketplace/tag.rb

```diff
@@ -5,6 +5,8 @@ class Tag < Record
     belongs_to :marketplace, inverse_of: :tags
     has_many :product_tags, inverse_of: :tag, dependent: :destroy
     has_many :products, through: :product_tags, inverse_of: :tags
+    has_many :menu_tag_products, -> { menu_tag_product }, through: :product_tags, source: :product, inverse_of: :tags
+    has_many :menu_products, -> { menu_product }, through: :product_tags, source: :product, inverse_of: :tags
```

Comment:
I see the `menu_tag_products` relationship used in this PR, but not `menu_products`.

I definitely do not want to use a relationship that uses a scope that includes eager-loading, that doesn't seem right to me.